server/GraknServer.java

```diff
@@ -127,6 +130,16 @@ private Path dataDir() {
         return command.dataDir();
     }
 
+    private void configureAndVerifyJavaVersion() {
+        int javaMajorVersion = Java.getMajorVersion();
+        if (javaMajorVersion == Java.UNKNOWN_VERSION) {
+            LOG.warn("Could not detect Java version from version string '{}'. Will start Grakn Core anyway.", System.getProperty("java.version"));
```

Comment:
Let's actually say `Will start Grakn Core Server anyway`

Reply:
@lolski addressed in https://github.com/graknlabs/grakn/pull/6251/commits/6958bb9122458f7786ca06b66786c338b46b1240